core/domain/prod_validation_jobs_one_off.py

```diff
@@ -356,6 +356,20 @@ class ExplorationSnapshotContentModelAuditOneOffJob(
         ProdValidationAuditOneOffJob):
     """Job that audits and validates ExplorationSnapshotContentModel."""
 
+    @classmethod
+    def enqueue(cls, job_id, additional_job_params=None):
+        """Marks a job as queued and adds it to a queue for processing.
+
+        Args:
+            job_id: str. The ID of the job to enqueue.
+            additional_job_params: dict(str : *) or None. Additional parameters
+                for the job.
+        """
+        # We have to decrease the number of shards for this job, since the job
+        # is getting killed due to Out Of Memory.
```

Comment:
Out Of Memory --> Out Of Memory errors

Reply:
According to the mail thread, OOM error, stopped occurring. Done.

Reply:
Not done? The text here has not been changed.

Reply:
I have removed it along with the function, since it didn't fail with any OOM errors.